client/angular.json

```diff
@@ -113,7 +113,7 @@
           }
         },
         "e2e": {
-          "builder": "@nrwl/cypress:cypress",
+          "builder": "@cypress/schematic:cypress",
           "options": {
             "cypressConfig": "cypress.json",
             "tsConfig": "cypress/tsconfig.json",
```

Comment:
See my comments in https://github.com/UMM-CSci-3601/3601-iteration-template/pull/712
```suggestion
```